### ISSUE TITLE
besselj_zero not exported

### ISSUE THREAD

**Issue by RainerHeintzmann** [chrisbrahms/Hankel.jl#35]
you may want to export `besselj_zero` or change your example in the documentations such that the function can be found.
It took me quite long to realize that this function is provided by this package, but not exported ;-)

**Comment by sethaxen** [comment]
Yes, we currently have a reference section, but we don't document the user-facing API, and we should.

@chrisbrahms, the API just includes the exported functions, correct?

**Comment by RainerHeintzmann** [comment]
It would already be quite helpful, if you specify `Hankel.besselj_zero` instead of `besselj_zero` in the example mentioned in the documentation.
